Source/FactoryGame/Public/Resources/FGTapeData.h

```diff
@@ -26,8 +26,11 @@ struct FSongData
 	UPROPERTY( EditDefaultsOnly, BlueprintReadOnly )
 	FName SongName;
 
-	/** Maximum song duration cached from the AkAudioEvent during cooking to allow accessing that information on Dedicated Servers */
-	UPROPERTY()
+	/** 
+ 	* Maximum song duration cached from the AkAudioEvent during cooking to allow accessing that information on Dedicated Servers
+  	* TODO(#372): revert it to non-editable in Blueprints once UFGTapeData::CachePlaylistSongDurations is reimplemented.
+  	*/
+	UPROPERTY( EditDefaultsOnly, BlueprintReadOnly )
```

Comment:
Would it be better to add it to default access transformers (Mods/Alpakit/Templates/BlueprintBlank/Config/AccessTransformers.ini)?

Reply:
Probably put it in SML's access transformers, as I don't think it makes sense for every mod created from the templates to have a copy of it

Reply:
Added to SML/Configs/AccessTransformers.ini, reverted the header change